interpreter/text/parser.mly

```diff
@@ -825,6 +834,14 @@ const_list :
   | /* empty */ { [] }
   | const const_list { $1 :: $2 }
 
+result :
+  | const { LitResult $1 @@ at () }
+  | LPAR CONST NAN RPAR { NanResult (nanop $2 ($3 @@ ati 3)) @@ at () }
+
+result_list :
+  | /* empty */ { [] }
+  | result result_list { $1 :: $2 }
```

Comment:
Should the result list be inside parens? Right now, when there are multiple return values specified, it seems like we would write `assert_return (invoke ...) result1 result2 result3)` but should it instead be  `assert_return (invoke ...) (result1 result2 result3))`? (Asking, not claiming it should be...)

Reply:
That's already the case in the current format. It is symmetric to the argument list, which is not being parenthesised either:
```
(assert_return (invoke "f" arg1 arg2 arg3) res1 res2 res3)
```
In fact, parens there would require inserting another keyword as head.